lib/tapioca/dsl/compilers/url_helpers.rb

```diff
@@ -1,13 +1,7 @@
 # typed: strict
 # frozen_string_literal: true
 
-begin
-  require "rails"
-  require "action_controller"
-  require "action_view"
-rescue LoadError
-  return
-end
+return unless defined?(Rails) && defined?(ActionDispatch::IntegrationTest) && defined?(ActionView::Helpers)
```

Comment:
Why the integration test? 🤔

Reply:
We have a direct reference to it [lower](https://github.com/Shopify/tapioca/pull/1765/files#diff-180fc447bbffd314ea8493148a280ac33a13b94c4006aacb339daaa97ec45b1dR99) but I wonder if the check could be moved there 🤔